packages/sdk/src/config/index.ts

```diff
@@ -1,25 +1,49 @@
 export const apiConfig = {
   version: "0.0.0.10",
   apis: {
-    mainnet: {
-      batter: "https://mainnet.ordit.io/",
-      orderbook: "1H4vvBnr62YWQmvNSt8Z4pDw3Vcv1n5xz7",
-      ipfs: "http://ipfs-gateway.ordit.io/"
+    bitcoin: {
+      mainnet: {
+        batter: "https://mainnet.ordit.io/",
+        orderbook: "1H4vvBnr62YWQmvNSt8Z4pDw3Vcv1n5xz7",
+        ipfs: "http://ipfs-gateway.ordit.io/"
+      },
+      regtest: {
+        batter: "https://regtest.ordit.io/",
+        orderbook: "bcrt1q2ys7qws8g072dqe3psp92pqz93ac6wmztexkh5",
+        ipfs: "http://ipfs-gateway.ordit.io/"
+      },
+      testnet: {
+        batter: "https://testnet.ordit.io/",
+        orderbook: "tb1qfnw26753j7kqu3q099sd48htvtk5wm4e0enmru",
+        ipfs: "http://ipfs-gateway.ordit.io/"
+      },
+      signet: {
+        batter: "https://signet.ordit.io/",
+        orderbook: "tb1qfnw26753j7kqu3q099sd48htvtk5wm4e0enmru",
+        ipfs: "http://ipfs-gateway.ordit.io/"
+      }
     },
```

Comment:
as we are not using the other options anymore, we can remove this and thin it out to

```suggestion
    bitcoin: {
      mainnet: "https://mainnet.ordit.io/",
      regtest: "https://regtest.ordit.io/",
      testnet: "https://testnet.ordit.io/",
      signet: "https://signet.ordit.io/",
    },
```

this will make migration easier as well for a 1:1 reference